frontends/election-manager/src/screens/smartcards_screen.tsx

```diff
@@ -297,3 +304,28 @@ export function SmartcardsScreen(): JSX.Element {
     </React.Fragment>
   );
 }
+
+export function SmartcardsScreen(): JSX.Element {
+  if (!areVvsg2AuthFlowsEnabled()) {
+    return <LegacySmartcardsScreen />;
+  }
+  return (
+    <NavigationScreen>
+      <Prose maxWidth={false}>
+        <h1>Smartcards</h1>
+        <h2>Election Cards</h2>
+        <p>
+          Insert a card to view card details and/or to create an{' '}
+          <strong>Admin or Poll Worker</strong> card for this election. The
+          election definition must be locked before Admin and Poll Worker cards
+          can be created.
+        </p>
+        <p>
+          <LinkButton small to={routerPaths.superAdminSmartcards}>
+            Create Super Admin Cards
+          </LinkButton>
```

Comment:
(Not sure if design feedback is helpful, so let me know if this has already been discussed/decided.)

When I first saw this page, it seemed like I should click the button. It wasn't clear til I read the fine print that inserting a card was the actual desired action. 

It seems like there should be a big loud prompt to insert a card instead of a button to create super admin cards.

Reply:
Very helpful! Beau's kept his guidelines fairly high-level here, with room for iteration.

I think that this is a great point. Let me make some tweaks, perhaps adding a graphic with a louder prompt to make the primary action (inserting a card) clearer

Reply:
Made some tweaks and posted updated screenshots in the PR description!